### ISSUE TITLE
Need to verify name of "View Creator Insights" permission

### ISSUE THREAD

**Issue by joe27g** [bender-devs/Bender-v4#3]
Generally, I add permission names directly from Discord for all languages, as they're guaranteed to match what's in the client. However, one of the newest permissions, `VIEW_CREATOR_MONETIZATION_ANALYTICS`, which I added in 891e974, was not available. I'm assuming this was because you have to enable server monetization to see it in the client. There are probably good ways to find this elsewhere, but I just guessed the name and added translations for it based on similar permissions and text in the Discord client.

**TL:DR** - The entry in all translation files, including English, for `PERMISSION_VIEW_CREATOR_MONETIZATION_ANALYTICS` needs to be verified.
